libnavigation-ui/src/test/java/com/mapbox/navigation/ui/internal/route/MapboxRouteLayerProviderTest.kt

```diff
@@ -1,5 +1,6 @@
 package com.mapbox.navigation.ui.internal.route
 
+import com.mapbox.navigation.ui.route.MapboxRouteLayerProvider
```

Comment:
Same as above re: move the test out of the `internal` package

Reply:
done